clang/lib/CConv/ConstraintBuilder.cpp

```diff
@@ -17,6 +17,43 @@
 using namespace llvm;
 using namespace clang;
 
+private int lastRecordLocation = -1;
+private void processInlineStruct(VarDecl *VD, ProgramInfo &Info, ASTContext *Context) {
```

Comment:
Seems to me you don't need this method anymore. In particular, you can just directly call `ConstraintResolver::constraintAllCVarsToWild` in the two places where you currently call this method, right?